twint/output.py

```diff
@@ -22,7 +22,10 @@ def _formatDateTime(datetimestamp):
     try:
         return int(datetime.strptime(datetimestamp, "%Y-%m-%d %H:%M:%S").timestamp())
     except ValueError:
-        return int(datetime.strptime(datetimestamp, "%Y-%m-%d").timestamp())
+        try:
+            return int(datetime.strptime(datetimestamp, "%d-%m-%Y %H:%M:%S").timestamp())
+        except ValueError:
+            return int(datetime.strptime(datetimestamp, "%Y-%m-%d").timestamp())
```

Comment:
I investigated this error and found that I was parsing the string incorrectly (my bad, I wasn't familiar with twint when I coded that), so If you simply replace `'%Y-%m-%d'` with `'%Y-%m-%d'` in tweet.py line 174, 176. users.py line 30. you won't need to add this line.
So what I previously did was parsed the string with a different format (dd-mm-yyyy) than what twint follows (yyyy-mm-dd) so it created some issue, so it's better to revert those parsed values to keep the date consistent throughout.

Reply:
Sorry have a bit of difficulty understanding " replace '%Y-%m-%d' with '%Y-%m-%d' " --

* For this one, sounds like we can just delete one of the lines ("%Y-%m-%d"?)

* Longer-term a better follow-on PR would be refactor datetime handling back to twint's convention of epoch_ms (I think) or datetime

Reply:
> Sorry have a bit of difficulty understanding " replace '%Y-%m-%d' with '%Y-%m-%d' " --
sorry my bad I it's supposed to be _replace '%d-%m-%Y' with '%Y-%m-%d'_ in the files I mentioned, that is, in tweet.py & users.py

Reply:
Ok updated to be Ymd-only and modified user/tweet.py to be Y-m-d from an exported settings table.

I'm iffier on changes here as `test.py` seems iffy ever since api break, and we're working through downstream fixes in our code bases in parallel as part of testing handling. But, pushed, in case you have better luck!